modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/open_window.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import getTop from './get_top';
+
+export default function openWindow(config, callback) {
+	const topUtil = getTop();
+
+	config.openingWindow = window;
+
+	topUtil.Liferay.Util._openWindowProvider(config, callback);
```

Comment:
@bryceosterhaus thoughts?

Reply:
> No, because it is defined in [frontend-js-aui-web/util.js](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js?rgh-link-date=2022-06-03T13%3A13%3A34Z#L1310-L1321)

Since this would fail if we disabled aui-web, we should probably find another solution since our goal is to be able to eventually deactivate that module entirely.

> replace its usages with openModal.

Are there any drawbacks in functionality to this approach?

Reply:
> Since this would fail if we disabled aui-web, we should probably find another solution since our goal is to be able to eventually deactivate that module entirely.

My thinking was to do this low-effort move and then as a scope of a different task replace the usages.

As you know, our modals are pretty complex so every usage (out of the 66) would need to be tested. It's not as simple as changing the name of the function.

And that would be done before we are at a point where we can turn off `frontend-js-aui-web`, because we still have a lot of work before disabling it (like modernizing/deprecating `Liferay.provide`).

Reply:
Yeah if we want to add this now and then wait to update until we tackle `Liferay.provide`, that is fine, but lets just be sure to make a note of it somewhere or create a ticket to make sure we don't forget about it.